config/environments/development.rb

```diff
@@ -101,4 +101,9 @@
 
   config.tophat_url = 'https://cdn.sit.earthdata.nasa.gov/tophat2/tophat2.js'
 
+  config.cmr_email_frequency = if ENV['cmr_email_frequency']
```

Comment:
I think you can us the safe nav operator here: `ENV['cmr_email_frequency']&.to_i || 3600`